components/Dropdown/Dropdown.js

```diff
@@ -2,24 +2,24 @@ class Dropdown {
   constructor(element) {
     
     // Assign this.element to the dropdown element
-    this.element;
+    this.element = element;
     
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
+    const button = document.querySelector('.dropdown-button');
     
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
+    this.content = document.querySelector('.dropdown-content');
+    // this.content = new TabItem(content);
+   
     
     // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
-
-    })
+    button.addEventListener('click', () => this.toggleContent());
   }
 
   toggleContent() {
     
     // Toggle the ".dropdown-hidden" class off and on
-    this.content;
```

Comment:
Here we should have toggled the class on `this.content` rather than on `this.element` because we want the content to hide/show when we click on the button. 👍